nodes/src/nodes/telegram/IEndpoint.py

```
@@ -0,0 +1,425 @@
+
+            except asyncio.CancelledError:
+                break
+            except Exception as e:
+                debug(f'Telegram polling error: {e}')
+                await asyncio.sleep(5)
+
+    # -------------------------------------------------------------------------
+    # Webhook route handler
+    # -------------------------------------------------------------------------
+
+    async def _webhook_handler(self, request):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_webhook_handler`

(ANN202)